recipes/quartet/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "quartet" %}
+{% set version = "1.0" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/ngodber/quartet/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: eaeebc506d743d59f04bb24725df5ed01d26775f8d94c73a70ab8838151a9303
+
+build:
+  number: 0
```

Does the package provide any ABI stability? This will need a `run_exports`, probably not and thus the following would be appropriate:
```suggestion
  number: 0
  run_exports:
    - {{ pin_subpackage("quartet", max_pin="x.x.x") }}
```